pandas/tests/indexing/test_categorical.py

```diff
@@ -46,6 +46,18 @@ def setup_method(self, method):
                 "B": (Series([1, 1, 2, 1, 3, 2]).astype(CDT([3, 2, 1], ordered=False))),
             }
         ).set_index("B")
+        self.df5 = DataFrame(
```

Comment:
rather than adding to something that needs to be changed (e.g. we need to paramterize these), create a fixture or just create in-line

Reply:
I've moved this to be created inline